### PR TITLE
Enforce flake8 in travis

### PR DESCRIPTION
**Pull request by theotherjimmy**

### CODE REVIEW

**Comment by theotherjimmy** [comment]
This first commit ought to go red real quick.

**Comment by theotherjimmy** [comment]
It's working

**Comment by bridadan** [comment]
Awesome! Let's be sure to add the expected format in the README when we get closer to merging this.

I'm guessing your intention would be to touch up all of the styling in either this PR or another one before this goes in?

**Comment by bridadan** [comment]
Oh boy, moving from `'` to `"`. Gotta break a lot of habits now 😛

**Comment by theotherjimmy** [comment]
@bridadan Better plan: set `git diff --name-only master | xargs black` as a pre-commit hook. Then do whatever.

**Comment by theotherjimmy** [comment]
That should do it. :crossed_fingers:

**Comment by theotherjimmy** [comment]
@bridadan Please review CI is green again!

**Comment by theotherjimmy** [comment]
I'll be adding a "Contributing.md" for the sake of explaining basically what I just did.

**Comment by bridadan** [comment]
Let's be sure to merge this after we come to a conclusion on #339 and #340 so we don't have to rebase those.

File: mbed_lstools/windows.py [hunk 1]
```diff
@@ -38,22 +36,25 @@
...
+MBED_STORAGE_DEVICE_VENDOR_STRINGS = [
+    "ven_mbed", "ven_segger", "ven_arm_v2m", "ven_nxp"
+]
 
 
 def _get_values_with_numeric_keys(reg_key):
     result = []
     try:
         for v in _iter_vals(reg_key):
+
```
Comment (bridadan) [inline mbed_lstools/windows.py]: When do we actually put extra lines in for `for` loops? This doesn't appear to be consistent within this file.

Ex. https://github.com/ARMmbed/mbed-ls/pull/342/files#diff-49012471517cafe043a3614156f872e7R226

File: mbed_lstools/windows.py [hunk 2]
```diff
@@ -142,65 +147,263 @@ def _determine_valid_non_composite_devices(devices, target_id_usb_id_mount_point
...
+                    target_id_usb_id,
+                )
+
+            vid_pid_target_id_usb_id_map[vid_pid_path][entry_key_string] = {
+                "target_id_usb_id": target_id_usb_id, "is_prefix": is_prefix
+            }
+    return vid_pid_target_id_usb_id_map
+
+
+def _update_important_bits(
```
Comment (bridadan) [inline mbed_lstools/windows.py]: Let's call this something a bit more descriptive. I'd suggest `_update_device_information` or `_update_device_details`.

**Comment by theotherjimmy** [comment]
Rebase time!

**Comment by bridadan** [comment]
This needs a little love, closing so we can revisit later.

**Pull request closed without merging**